### ISSUE TITLE
error with serial port module - VSCode

### ISSUE THREAD

**Issue by patrickchan00** [pycom/Pymakr#28]
error message "There was an error with your serial port module, Pymakr will likely not work properly...."  shows up when starting visual studio code. Tried to reinstall but doesn't work.

VSCode:
Version: 1.59.0 (user setup)
Commit: 379476f0e13988d90fab105c5c19e7abc8b1dea8
Date: 2021-08-04T23:13:12.822Z
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Windows_NT x64 10.0.19042

Pymakr:
v1.1.12

**Comment by yucheng1993** [comment]
Yes, we used this vsvode version too and it returned back the simialr issue. So we apply the ATOM to continue our development work.

**Comment by claudiorodr** [comment]
Experienced the same issue after updating VSCode. Either revert the update or move to Atom. Hope Pycom team will be fixing this anytime soon..

**Comment by Bliph** [comment]
Probably the same issue as https://github.com/pycom/pymakr-vsc/issues/144

**Workaround:** Downgrade VSCode to 1.58.x
